render-test/runner.hpp

```diff
@@ -14,6 +14,11 @@ class TestRunner {
     bool run(TestMetadata&);
     void reset();
 
+    /// Returns path of the render tests root directory.
+    static const std::string& getBasePath();
+    /// Returns path of mapbox-gl-native expectations directory.
+    static const std::string& getPlatformExpectationsPath();
+
```

Comment:
This could return a list of paths that the test runner would search at the order these paths are inserted.

The fallback (platform expectations) would be given to the test runner as parameter.

Reply:
Thanks for the proposal! done in the latest commit.